src/hobbygroups/less/hobbygroups.less

```diff
@@ -9,26 +10,44 @@
 .hobby {
   display: grid;
   grid-template-columns: 1fr 2fr;
-  .owCard();
+  justify-items: center;
 
   & img {
-    width: 100%;
+    width: 11rem;
+    align-self: center;
+    z-index: 1;
+    .owCard();
   }
 
   & > div {
-    margin: 20px;
+    padding: 20px;
+    .owCard();
   }
 }
 
 .hobbyTitle {
   margin-bottom: 10px;
 }
 
-@media only screen and (max-width: 1024px) {
+@media only screen and (max-width: @owDesktopMaxWidth) {
   .container {
     grid-template-columns: 1fr;
     grid-template-rows: auto;
   }
+  .hobby {
+    grid-template-columns: 1fr;
+
+    & > div {
+      margin-top: 5px;
+      width: 40rem;
+    }
+  }
+}
+
+@media only screen and (max-width: @owMobileBreakpoint) {
```

Comment:
Okay so I really want for there to be a dedicated tablet mode, because it changes to the mobile-view way too quickly at the moment. Like, if I open the devtools, I still want it to be in desktop-mode